test/gtest/ucp/test_ucp_wireup.cc

```diff
@@ -1149,6 +1205,60 @@ class test_ucp_wireup_fallback_amo : public test_ucp_wireup {
     }
 };
 
+class test_ucp_wireup_amo : public test_ucp_wireup {
+public:
+    static ucp_params_t get_ctx_params() {
+        ucp_params_t params = test_ucp_wireup::get_ctx_params();
+        params.field_mask  |= UCP_PARAM_FIELD_REQUEST_SIZE;
+        params.request_size = sizeof(test_ucp_wireup_amo *);
+        return params;
+    }
+
+    static std::vector<ucp_test_param>
+    enum_test_params(const ucp_params_t& ctx_params, const std::string& name,
+                     const std::string& test_case_name, const std::string& tls)
+    {
+        return enum_test_params_features(ctx_params, name, test_case_name, tls,
+                                         UCP_FEATURE_AMO32 | UCP_FEATURE_AMO64,
```

Comment:
can we require only UCP_FEATURE_AMO32/UCP_FEATURE_AMO64, depending on sizeof(elem_type) ?